### PR TITLE
wholeBodyDynamics: add feature to use MAS interface based  RFE IMU 

### PR DESCRIPTION
**Pull request by prashanthr05**
Only if the specific group is present in configuration file, the relevant feature is enabled.
Otherwise, fully compatible with default behavior.

### CODE REVIEW

File: devices/wholeBodyDynamics/README.md [hunk 1]
```diff
@@ -46,6 +46,9 @@ For an overview on `wholeBodyDynamics` and to understand how to run the device,
 |                      | enableGravityCompensation | bool | -  | -           | No        |  |  |
 |                      | gravityCompensationBaseLink| string | - | -         | No        | ..  | |
 |                      | gravityCompensationAxesNames | vector of strings | - | - | No   | Axes for which the gravity compensation is published. | |
+| HW_USE_MAS_IMU |  -  | group             | -     | -     | No       |  Group to enable attach to Multiple Analog Sensor interface based IMU. | If the group is not present, the default behavior of attaching to IGenericSensor interface IMu will be carried out. |
```
Comment (traversaro) [inline devices/wholeBodyDynamics/README.md:49]:
```suggestion
| HW_USE_MAS_IMU |  -  | group             | -     | -     | No       |  Group to enable attach to Multiple Analog Sensor interface based IMU. | If the group is not present, the default behavior of attaching to IGenericSensor interface IMU will be carried out. |
```

**Comment by traversaro** [comment]
cc @Nicogene 

I guess eventually this should become the default. Does it make sense to put this in the CHANGELOG?

**Review by traversaro — CHANGES_REQUESTED** [review]
Minor comment.

File: devices/wholeBodyDynamics/README.md (hunk 1 above)
Comment (prashanthr05) [reply devices/wholeBodyDynamics/README.md]: Fixed in https://github.com/robotology/whole-body-estimators/pull/99/commits/5dd8c48392d5abf0e48500c4870540663db0cbec

**Comment by prashanthr05** [comment]
> Does it make sense to put this in the CHANGELOG?

Indeed. I updated CHANGELOG in https://github.com/robotology/whole-body-estimators/pull/99/commits/5dd8c48392d5abf0e48500c4870540663db0cbec

**Comment by prashanthr05** [comment]
friendly ping @traversaro !

**Review by traversaro — APPROVED** [review]

**Comment by traversaro** [comment]
Thanks @prashanthr05 , sorry for the delay.

**Comment by prashanthr05** [comment]
Thank you @traversaro . merging.

**Pull request merged**